modules/temp-channels/channel-settings.js

```diff
@@ -0,0 +1,282 @@
+    let allowedUsers = vc.allowedUsers.split(',');
+    let removedUser = null;
+    if (callerInfo === 'command') {
+        removedUser = interaction.options.getUser('user');
+    }
+    if (callerInfo === 'modal') {
+        const removedUserString = interaction.fields.getTextInputValue('remove-modal-input');
+        try {
+            removedUser = interaction.guild.members.cache.find(member => member.user.tag === removedUserString).user;
+        } catch (e) {
+            try {
+                removedUser = await client.users.fetch(removedUserString);
+            } catch (f) {
+                interaction.editReply(localize('temp-channels', 'user-not-found'));
+                return;
+            }
+        }
+    }
+    const user = removedUser.id;
+    allowedUsers = allowedUsers.filter((e => e !== user));
+    allowedUsers = allowedUsers.toString();
+    vc.allowedUsers = allowedUsers;
+    await vc.save();
+    const vchann = interaction.guild.channels.cache.get(vc.id);
+    try {
+        await vchann.permissionOverwrites.delete(removedUser);
+    } catch (e) {
+        console.log(e);
+    }
+    const usr = interaction.guild.members.cache.get(removedUser.id);
+    if (usr.voice.channelId === vc.id) {
+        try {
+            await usr.voice.disconnect();
+        } catch (e) {
+            interaction.editReply(localize('temp-channels', 'no-disconnect'));
+            return;
+        }
+    }
+    interaction.editReply(await embedType(moduleConfig['userRemoved'], {'%user%': removedUser.tag}, {ephemeral: true}));
+};
+
+module.exports.usersList = async function (interaction) {
+    const moduleConfig = interaction.client.configurations['temp-channels']['config'];
+    const vc = await client.models['temp-channels']['TempChannel'].findOne({
+        where: {
+            [Op.and]: [
+                {id: interaction.member.voice.channelId},
+                {creatorID: interaction.member.id}
+            ]
+        }
+    });
+    const allowedUsersArray = vc.allowedUsers.split(',');
+    let allowedUsers = '';
+    for (const user of allowedUsersArray) {
+        allowedUsers = allowedUsers + '\n • <@' + user + '>';
+    }
+    if (allowedUsersArray.at(0) === '') {
+        interaction.editReply(localize('temp-channels', 'no-added-user'));
+        return;
+    }
+    interaction.editReply(moduleConfig['listUsers'] + ' ' + allowedUsers);
+};
+
+module.exports.channelEdit = async function (interaction, callerInfo) {
+    const moduleConfig = interaction.client.configurations['temp-channels']['config'];
+    const vc = await client.models['temp-channels']['TempChannel'].findOne({
+        where: {
+            [Op.and]: [
+                {id: interaction.member.voice.channelId},
+                {creatorID: interaction.member.id}
+            ]
+        }
+    });
+    const vchann = interaction.guild.channels.cache.get(vc.id);
+    let edited = 0;
+    let vcNsfw;
+    let vcBitrate;
+    let vcLimit;
+    let vcName;
```

Comment:
These values should be set to the current value of the channel, otherwise errors like `user_limit: Value "" is not int` might appear if you only try to edit one field

Reply:
This might be a problem with Discords new strict [type enforcement](https://discord.com/channels/613425648685547541/697138785317814292/1054837953752486018), as our friend JavaScript just assumes that you mean a string